__tests__/integration.spec.js

```diff
@@ -85,6 +86,52 @@ describe('toMatchImageSnapshot', () => {
     });
   });
 
+  describe('updates', () => {
+    const customSnapshotIdentifier = 'integration-update';
+    const updateImageData = fs.readFileSync(fromStubs('TestImageUpdate1pxOff.png'));
+    const updateImageSnapshotPath = path.join(__dirname, '__image_snapshots__', `${customSnapshotIdentifier}-snap.png`);
+
+    beforeEach(() => {
+      fs.writeFileSync(updateImageSnapshotPath, imageData);
+    });
+
+    afterAll(() => {
+      fs.writeFileSync(updateImageSnapshotPath, imageData);
+    });
+
+    it('does not write a result image for passing tests in update mode by default', () => {
+      const updateModeMatcher = toMatchImageSnapshot.bind({
+        snapshotState: new SnapshotState(__filename, {
+          updateSnapshot: 'all',
+        }),
+        testPath: __filename,
+      });
+      updateModeMatcher(updateImageData, {
+        customSnapshotIdentifier,
+        failureThreshold: 2,
+        failureThresholdType: 'pixel',
+      });
+      expect(fs.readFileSync(updateImageSnapshotPath)).toEqual(imageData);
+    });
+
+    it('writes a result image in update mode with updatePassedSnapshots: true', () => {
```

Comment:
for passing tests right? Can you clarify that in the test name?

Reply:
Fixed.